layouts/_default/staff-page.html

```diff
@@ -1,3 +1,77 @@
+{{ define "body-class" }}
+  text-[16px] bg-beige font-serif leading-normal text-s-9
+{{ end }}
+
+{{ define "static-head" }}
+  {{ partialCached "header/static-tw.html" . }}
+{{ end }}
+
+{{ define "header" }}
+  <header data-ga-category="header">
+    {{ block "nav" . }}
+      {{ partialCached "tw/nav.html" . }}
+    {{ end }}
+    {{ block "banner" . }}
+      {{ if .Param "banner-active" }}
+        {{ partialCached "tw/banner.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "topper" . }}
+      {{ if .Param "topper-active" }}
+        {{ partialCached "tw/promo-topper.html" . }}
+      {{ end }}
+    {{ end }}
+    {{ block "promo" . }}
+      {{ partialCached "tw/promo-header.html" . }}
+    {{ end }}
+  </header>
+{{ end }}
+
+{{ define "main" }}
+  <header class="bg-black py-14 px-3 text-white">
+    <div
+      class="mx-auto max-w-[1248px] items-center md:min-h-[110px] gap-y-3 gap-x-5 md:grid md:grid-cols-12"
+    >
+      <h1
+        class="uppercase font-serif text-[45px] leading-none col-span-5 col-start-2 row-span-1 row-start-1 font-black"
```

Comment:
The classes aren't being sorted by prettier because it thinks this is a Bulma page. Add _default/staff-page.html to line 23 of prettier.config.js.

Also `font-black` is an ultrabold. I find it confusing, because you would think it's just the same as `text-black`. Maybe it would be more clear if we used `font-[900]` instead? Up to you.

Reply:
I've gotten used to setting 900 weight with font-black ATP so I feel good about keeping it.